src/CBF/CBF.jl

```diff
@@ -385,12 +403,14 @@ function MOI.read_from_file(model::Model, filename::String)
                 for k in 1:parse(Int, strip(readline(io)))
                     raw_coord = split(strip(readline(io)))
                     @assert length(raw_coord) == 4
-                    (psd_var_idx, i, j) = (parse(Int, raw_coord[i]) + 1 for i in 1:3) # CBF indices start at 0.
+                    (psd_var_idx, i, j) = (parse(Int, raw_coord[i]) + 1
```

Comment:
`(psd_var_idx, i, j) = parse.(Int, raw_coord[1:3]) .+ 1`?

Below also

Reply:
wouldn't that have slightly worse performance (eg allocating temp vector for raw_coord[1:3])?